sjsonnet/src/sjsonnet/Val.scala

```diff
@@ -469,32 +470,25 @@ object Val{
   }
 
   /** Superclass for standard library functions */
-  abstract class Builtin(paramNames: Array[String], defaults: Array[Expr] = null)
+  abstract class Builtin(val functionName: String, paramNames: Array[String], defaults: Array[Expr] = null)
```

Comment:
A nice consequence of this new `functionName` thing is that we can now straightforwardly improve the `RunProfiler` output for built-in functions.

Today this outputs confusing anonymous class names like

```
[info] - 1029ms 762549  1350ns  sjsonnet.Std$$anon$8
[info] - 153ms  499763  307ns   sjsonnet.Std$$anon$5
```

but if we change

https://github.com/databricks/sjsonnet/blob/89a83b1dcf9ee5206be04525f25477ae50ad5f92/bench/src/main/scala/sjsonnet/ProfilingEvaluator.scala#L160-L161

to use `.functionName` then we get nice output like

```
[info] - 914ms  762032  1200ns  setMember
[info] - 133ms  233603  572ns   stripChars
```